### ISSUE TITLE
Import of Custom Schema Query / Mutation definition fails

### ISSUE THREAD

**Issue by ma-jahn** [ardatan/graphql-import#395]
Hi,

i'm trying to integrate eslint-plugin-graphql to validate my queries against and SDL File.
Unfortunately the import fails since we have a custom schema definition.

```
schema {
  query: CustomerQuery
  mutation: CustomerMutation
}

type CustomerQuery {
  // All our queries
}

type CustomerMutation {
  // All our Mutations
}

type Mutation {
 // an independent type for Changes on certain objects
}
```

When importing the sdl i receive the following

```
SCHEMA GraphQLSchema {
  __validationErrors: undefined,
  extensions: undefined,
  astNode: undefined,
  extensionASTNodes: undefined,
  __allowedLegacyNames: [],
  _queryType: undefined,
  _mutationType: Mutation,
  _subscriptionType: undefined,
 ...
}
```

As you can see the schema definition is not being considered when building the GraphQLSchema. Our `Mutation` Type is being used as _mutationType instead of the actual`CustomerMutation`.

Am i doing something wrong or is this simply not supported?
Thank you!

**Comment by ardatan** [comment]
Hi @ma-jahn !
In 1.0.0 beta release, we introduced a lot of changes including schema definition support;
Could you install `graphql-import@beta` to try new changes? Don't forget to modify your code regarding to the migration notes in `README`. 
https://github.com/ardatan/graphql-import#updating-from-07x

**Comment by ardatan** [comment]
Available in 1.0.0!
